app/src/main/java/com/trongntd/themovie/entity/Video.java

```diff
@@ -0,0 +1,11 @@
+package com.trongntd.themovie.entity;
```

Comment:
Never use the private name in a project. ```trongntd``` is something we should get rid of.

Reply:
Yup, I will fixed on current branch working. Thanks